### PR TITLE
Added FBPrintWindow command

### PR DESCRIPTION
**Pull request by PeteTheHeat**
I've been working with input accessory views quite a bit. They've been annoying to debug because they don't live in the main key window, they are hosted in the Keyboard's window. I added a short chisel command locally to make printing a window easy. Let me know if this is something useful for the library.

### CODE REVIEW

**Comment by kastiglione** [comment]
Thanks Pete. I like it. I have some thoughts.

This is the same as `pviews`, except `pviews` is hard coded to use `keyWindow`. What do you think about making this a flag on `pviews`? For example: `pviews -w 0`, `pviews -w 1`, etc.

**Comment by kastiglione** [comment]
@PeteTheHeat what do you think about my suggestion?

**Comment by PeteTheHeat** [comment]
Yeah that makes sense! Let me update it.

File: commands/FBPrintCommands.py [hunk 1]
```diff
@@ -52,20 +52,27 @@ def options(self):
...
 
   def run(self, arguments, options):
     maxDepth = int(options.depth)
+    window = int(options.window)
     isMac = runtimeHelpers.isMacintoshArch()
 
-    if arguments[0] == '__keyWindow_dynamic__':
-      arguments[0] = '(id)[[UIApplication sharedApplication] keyWindow]'
-
+    if window > 0:
```
Comment (kastiglione) [inline commands/FBPrintCommands.py]: It looks like this check can be removed.

Comment (kastiglione) [reply commands/FBPrintCommands.py]: I see now why you have this. Never mind :)

File: commands/FBPrintCommands.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 def lldbcommands():
   return [
     FBPrintViewHierarchyCommand(),
+    FBPrintWindow(),
```
Comment (kastiglione) [inline commands/FBPrintCommands.py]: Now that `pviews` supports `-w`, I don't think it's worth having a separate command. A user could get `pwindows` by adding this to their `~/.lldbinit`:

```
command alias pwindow pviews --window %1
```

File: commands/FBPrintCommands.py (hunk 1 above)
Comment (PeteTheHeat) [reply commands/FBPrintCommands.py]: Yup I was hoping the if/else flow was clear, if a window is passed in, do some new stuff, else fall back to what the code was doing before.

File: commands/FBPrintCommands.py (hunk 2 above)
Comment (PeteTheHeat) [reply commands/FBPrintCommands.py]: Yup, I tried to delete it but it came back in the merge somehow lol, I'm too used to hg these days. I'll remove it

**Comment by kastiglione** [comment]
thanks!

**Pull request merged**